Automattic/pocket-casts-android
# Custom Playback Effects: Hide segmented control for local files

## Description
This hides effects settings segmented control for local files.

Part of #3042

## Testing Instructions
1. Play a local file 
2. Open effects view from full-screen player
3. ✅ Notice that  effects settings segmented control is not present
4. Play a podcast file
5. Open effects view from full-screen player
6. ✅ Notice that  effects settings segmented control is present

## Screenshots or Screencast 
https://github.com/user-attachments/assets/5822a3cf-7ba9-41d9-b3c5-12341fa866d0

## Checklist
 
#### I have tested any UI changes...
